tests/stress/sync_stress/test_sync.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+import time
+from pytest_testconfig import config as testconfig
+
+from tests.test_bs import add_multi_clients, get_conf, setup_bootstrap, save_log_on_exit
+import tests.queries as q
+from tests.convenience import convert_ts_to_datetime
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+SYNC_DONE = "sync done"
+START_SYNC = "start synchronize"
+
+
+def test_sync_stress(init_session, setup_bootstrap, save_log_on_exit):
+    bs_info = setup_bootstrap.pods[0]
+    cspec = get_conf(bs_info, testconfig['client'])
+
+    clients = add_multi_clients(init_session, cspec, 10)
+
+    hits = []
+    number_of_pods = len(clients) + 1  # add 1 for bootstrap pod
+    tts = 70
+    while len(hits) != number_of_pods:
+        print(f"waiting for all clients to finish downloading all files, sleeping for {tts} seconds")
+        time.sleep(tts)
+        hits = q.get_all_msg_containing(init_session, init_session, "Done downloading")
+
+    del cspec.args['remote-data']
+    cspec.args['data-folder'] = ""
+
+    # Adding a new client
+    res_lst = add_multi_clients(init_session, cspec, 1, 'client')
+    new_client = res_lst[0]
+
+    curr_try = 0
+    max_retries = 20
+    interval_time = 60
+    print("waiting for new pod to be synced")
+    while True:
+        hits = q.get_all_msg_containing(init_session, new_client, SYNC_DONE, is_print=False)
+        if hits:
+            break
+
+        print(f"not synced after {curr_try} tries of {interval_time} secs each", end="\r")
+        time.sleep(interval_time)
```

Comment:
sleep for 1 minute?

Reply:
yes.. do you think it should be a diff value?

Reply:
It depends on how long the entire sync is done, if it takes 5 sec I wouldn't wait a minute vefore polling again

Reply:
it takes between 12-18 mins the whole syncing process.
sleeping for 1 min shows process as well. it doesn't look stuck.
not critical to me. I can change it to 2 mins if you'd like